### PR TITLE
016/master

### LINKED ISSUE #15: Update tests
When we update to rails 5.0, there will be a deprecation warning on tests that don't use named parameters. Tests should be updated accordingly. 

example
```diff
-        post '/sign-in', credentials: user_params
+        post '/sign-in', params: { credentials: user_params }
```

### PR DESCRIPTION
**Pull request by raq929**
In combination with 016/solution closes #15

### CODE REVIEW

**Comment by raq929** [comment]
I haven't updated to close #7 and #15, and I'd appreciate your input on those, and then I can update this PR.

File: README.md [hunk 1]
```diff
@@ -46,6 +48,29 @@ What do our user stories do for the scope of our project?
 How are they useful when wireframing our webapp's layout and UX?
 How do good user stories and wireframes help with app development?
 
+## Behavior Driven vs. Test Driven Development
+Tests can be written before or after writing development code. Writing tests
+after development is called 'backfilling'.  Test driven development (TDD) is a
+specific order of testing and writing code:
+1. Write a test
+2. Run the test (it should fail)
```
Comment (jrhorn424) [inline README.md]: Markdown numbering, please

File: README.md [hunk 2]
```diff
@@ -46,6 +48,29 @@ What do our user stories do for the scope of our project?
...
+4. Run the test (if it fails, go back to step 3)
+5. Refactor
+6. Run the test
+
+It often refers to bottom-up testing, in which unit tests are written first, and
+features are tested afterwards using integration tests. TDD is a challenge, and
+if you're feeling up for it, try it! Backfilling tests may be a more attainable
+goal for your project.
+
+Behavior Driven Development is top-down testing. g. It can be done either before
```
Comment (jrhorn424) [inline README.md]: Dangling "g"

Comment (jrhorn424) [inline README.md]: :joy:

**Pull request merged**